docs/modules/ROOT/pages/reference/index.adoc

```diff
@@ -56,9 +56,91 @@
 |Stable
 |1.5.11
 |
+
```

Comment:
```suggestion
```

Could you please remove this change? The Part between standards: START and standards: END is generated at build time, so it gets removed and therefore the CI complains about unexpected changes in the source tree after the build. 

Running `mvn -f docs/ -am generate-resources` would do the same.